services/rag/Dockerfile

```
@@ -74,13 +74,17 @@ FROM python:3.11-slim AS runtime
 WORKDIR /app
 
 # Install ONLY runtime dependencies (no build-essential, no libpq-dev)
-# - curl: Used by health check
+# - curl: Used by health check + dbmate binary download
 # - libpq5: PostgreSQL runtime library (not the dev headers)
 # Note: SOPS binary is already provided by COPY --from=builder /usr/local/bin
+ARG DBMATE_VERSION=2.20.0
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)